HoaryFox/KarambaConnect/CrossSection.cs

```diff
@@ -0,0 +1,195 @@
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using Karamba.CrossSections;
+using Karamba.Materials;
+using STBReader;
+using STBReader.Member;
+using STBReader.Section;
+
+namespace HoaryFox.KarambaConnect
+{
+    public class CrossSection
+    {
+        public static List<string>[] GetIndex(StbData stbData)
+        {
+            var k3Ids = new List<string>[2];
+
+            var frameList = new List<StbFrame> { stbData.Columns, stbData.Girders, stbData.Braces };
+            foreach (StbFrame frame in frameList)
+            {
+                int index = frame is StbBraces ? 1 : 0;
+                for (var eNum = 0; eNum < frame.Id.Count; eNum++)
+                {
+                    k3Ids[index].Add("Id" + frame.IdSection[eNum]);
+                }
+            }
+
+            return k3Ids;
+        }
+        
+        public static List<CroSec> GetCroSec(StbData stbData)
+        {
+            // TODO: 材軸の回転は未設定（どこで設定するかも謎）
+            double p1, p2, p3, p4;
+            string name;
+            ShapeTypes shapeType;
```

Comment:
Codacy found an issue: [Remove the unused local variable 'shapeType'.](https://app.codacy.com/gh/hrntsm/HoaryFox/pullRequest?prid=6405207)

Reply:
fix